backend/src/meetings/entities/comment.entity.ts

```diff
@@ -16,9 +15,12 @@ export class CommentEntity {
     @CreateDateColumn()
     creationDate: Date;
 
-    @ManyToOne(() => UserEntity, {eager: true})
+    @ManyToOne(() => UserEntity)
```

Comment:
, {eager: true}) was removed. Why? In this case it helps us because we can be 100% sure that the user object will be loaded. For example, the toDTO Method could cause a null pointer now.